src/Features/Core/Portable/EditAndContinue/EditAndContinueDiagnosticDescriptors.cs

```diff
@@ -166,7 +166,7 @@ void AddGeneralDiagnostic(EditAndContinueErrorCode code, string resourceName, Di
         AddGeneralDiagnostic(EditAndContinueErrorCode.ErrorReadingFile, nameof(FeaturesResources.ErrorReadingFile));
         AddGeneralDiagnostic(EditAndContinueErrorCode.CannotApplyChangesUnexpectedError, nameof(FeaturesResources.CannotApplyChangesUnexpectedError));
         AddGeneralDiagnostic(EditAndContinueErrorCode.ChangesDisallowedWhileStoppedAtException, nameof(FeaturesResources.ChangesDisallowedWhileStoppedAtException));
-        AddGeneralDiagnostic(EditAndContinueErrorCode.DocumentIsOutOfSyncWithDebuggee, nameof(FeaturesResources.DocumentIsOutOfSyncWithDebuggee));
+        AddGeneralDiagnostic(EditAndContinueErrorCode.DocumentIsOutOfSyncWithDebuggee, nameof(FeaturesResources.DocumentIsOutOfSyncWithDebuggee), DiagnosticSeverity.Warning);
```

Comment:
curious what determines if something is just a warning or not.

Reply:
Warning doesn't block applying change. We don't report this anymore since we don't have necessary info from project system. Follow up: https://github.com/dotnet/roslyn/issues/78125

Reply:
thanks!